plugins/main/public/components/common/welcome/welcome.scss

```diff
@@ -1,14 +1,3 @@
-.wz-welcome-page .euiCard .euiTitle,
-.wz-module-body .euiCard .euiTitle {
-  font-size: 16px;
-  font-weight: 400;
-}
-
-.wz-welcome-page .euiCard .euiText,
-.wz-module-body .euiCard .euiText {
-  font-family: sans-serif;
```

Comment:
**question**: Is unsetting the font-family styles intentional?

Reply:
The classes `.wz-welcome-page` and `.wz-module-body` are no longer in use.